ess/src/API/EMBC.ESS/Resources/Cases/Evacuations/EvacuationRepository.cs

```diff
@@ -357,6 +357,24 @@ public async Task<string> Update(EvacuationFile file)
             return updatedEvacuationFile.era_name;
         }
 
+        public async Task<string> UpdateSecurityPhrase(string essFileNumber, string securityPhrase)
+        {
+            var evacuationFile = essContext.era_evacuationfiles
+                .Where(f => f.era_name == essFileNumber)
+                .ToArray()
+                .FirstOrDefault();
+
+            if (evacuationFile == null) return null;
```

Comment:
rename essFileNumber to fileId and probably a good idea to throw an exception if not found